src/components/Books.js

```diff
@@ -1,18 +1,26 @@
 import BookList from './Booklist';
 import NewBook from './NewBook';
+import styles from '../styles/Books.module.css';
 
 function Books() {
   return (
     <>
       <div style={{
         display: 'flex',
-        flexDirection: 'row',
+        flexDirection: 'column',
         justifyContent: 'center',
         alignItems: 'center',
+        columnGap: '20px',
       }}
```

Comment:
- [` OPTIONAL`] Kindly consider adding a `background-color` to the main body of your web app as shown in the original design. This will help make your content stand out to help create a good user experience. See below for more details:

<table>
<tr>
<th align=“middle”>Replin design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="703" alt="Screenshot 2023-07-20 at 15 47 28" src="https://github.com/Kidd254/Bookstore-React-Redux/assets/107192076/30ebab2a-d2ec-4168-a191-e62b91eb3147">
</td>
<td>
<img width="1440" alt="Screenshot 2023-07-20 at 16 29 35" src="https://github.com/Kidd254/Bookstore-React-Redux/assets/107192076/4ff95fd7-7ae3-4484-a4f7-7ea47786c73d">
</td>
</tr>
</table>